_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+
+## The parts of the whole
+
+To see how this works, let’s first look at some of the components in Security Analytics.
+
+### Detectors
+
+Central to Security Analytics are its detectors. Detectors are components that interpret log data, compare that data to a schema of rules, and define the triggers that generate findings and the alerts that notify administrators of possible security events in the system. Their advantage is that they offer flexibility through their configuration, and this allows a business to fine tune its detection capabilities for specific threats, wider objectives, or targets in between.
+
+### Rules
+
+Security Analytics leverages the versatility, portability, and up-to-date framework of [Sigma rules](https://github.com/SigmaHQ/sigma). These rules define the conditional logic applied to ingested log data and allow the system to identify an event of interest. In general, they determine exactly what type of event or attack the system is looking out for. Security Analytics uses these prepackaged, open-source rules as a starting point for describing relevant log events. But with their inherently flexible format and wide compatibility, users can import and customize Sigma rules to meet their needs.
+
+### Log types
+
+Log data is key to understanding activity in any network. Whether it’s server logs, firewall logs, infrastructure logs, application logs, or endpoint logs, this data is elemental in gaining insight into the activity within a network. In terms of security coverage, the more log data a security infrastructure can process the better. Similarly, the wider the variety of log types that infrastructure can ingest, the more versatile that solution. Security Analytics launched with options for the most common log types. Since then, we have devoted significant energy to expanding support for more and more log types. And that trend will continue.
+
+### The user interface and dashboards
+
+The Security Analytics user interface is part of OpenSearch Dashboards and provides real-time visualizations for findings, alerts, and other analytics. Interaction with the Dashboards UI is intuitive and lets users easily navigate features and perform essential actions. Most importantly, the interface presents graphical information on network events happening in the moment. And to facilitate analysis of findings, we are continuously expanding support for dashboards that can present findings in a number of varied formats—such as charts, graphs, and maps—giving users the ability to customize their visualizations.
```

Comment:
"The Security Analytics user interface is part of OpenSearch Dashboards"

This gives the impression that only Dashboards has support for this feature, when instead this can be entirely used with the rich set of APIs included.